public/bell-icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="23" height="24" viewBox="0 0 23 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path d="M4.15156 15.7125L7.67578 19.2462C7.44587 19.485 7.17085 19.6755 6.86677 19.8065C6.56269 19.9375 6.23564 20.0065 5.9047 20.0094C5.57376 20.0123 5.24557 19.949 4.93926 19.8233C4.63296 19.6976 4.35468 19.512 4.12067 19.2772C3.88665 19.0425 3.70158 18.7634 3.57626 18.4561C3.45094 18.1489 3.38788 17.8197 3.39076 17.4877C3.39364 17.1558 3.46239 16.8277 3.59301 16.5227C3.72364 16.2177 3.91351 15.9419 4.15156 15.7112V15.7125ZM15.2015 22.3675L14.3205 23.25L0.238525 9.12499L1.11833 8.24124L2.49164 8.43749C3.43126 8.57249 4.59769 8.28499 5.07997 7.79999L9.92141 2.94499C11.3265 1.56214 13.219 0.791616 15.1875 0.800988C17.1559 0.81036 19.0411 1.59887 20.433 2.99504C21.8249 4.39121 22.611 6.28213 22.6203 8.2566C22.6297 10.2311 21.8615 12.1294 20.4829 13.5387L15.6414 18.3962C15.1579 18.8812 14.8713 20.05 15.0059 20.9925L15.2015 22.3675Z" fill="white"/>
+</svg>
```

Comment:
There is a small trick (that I learn from Pae I think :D ) with Figma:
If you export the SVG by selecting the parent container of the icon, then you would get an icon with the right dimensions (including the padding!!) and centered :) 

This would reduce the need for manuall CSS adjustment:

|1. Select the parent container in figma | 2. Export for SVG |
|---|---|
| ![figma tip](https://user-images.githubusercontent.com/77609814/160774882-3581952a-b386-4e52-8378-b7142296f0cb.gif) |![image](https://user-images.githubusercontent.com/77609814/160775039-b27a27a1-753f-489e-b0f9-b05d077880f7.png)|

3. The resulting SVG is easier to position (paddings are already there):

![image](https://user-images.githubusercontent.com/77609814/160774615-d0d4e6ab-8ef7-4851-9dbf-83e66fb5173e.png)

Reply:
that's a nice little trick right there 🚀